### PR TITLE
Flesh out serological tests

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

I tried to fill this info in a little more! It is not my area of expertise, though, so please feel free to change as you see fit!

### Suggested reviewers (optional)

@nilswellhausen 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/10.diagnostics.md [hunk 1]
```diff
@@ -331,18 +331,56 @@ Further research is required to determine the full extent to which sustained imm
...
+Instead of using an enzyme-labeled antibody as a probe that binds to the target antigen, instead the probe is an antigen and the target is an antibody.
+On March 18, 2020, the Krammer lab of the Icahn School of Medicine at Mount Sinai proposed an ELISA test that detects IgG and IgM that react against the RBD of the spike proteins (S) of the virus [@doi:10.1101/2020.03.17.20037713; @doi:10.1038/s41591-020-0913-5].
+A subsequent ELISA test developed to detect SARS-CoV-2 IgG based on the RBD have reported a selectivity of over 99% and a sensitivity of up to 88.24%, which was observed in samples collected 21 to 27 days after the onset of infection (approximated with symptom onset or positive PCR test) [@doi:10.3389/fmicb.2020.618097].
+Earlier in the disease course, sensitivity was lower: 53.33% between days 0 and 13 and 80.47% between days 14 and 20.
+This study reported that their laboratory ELISA outperformed two commercial kits that also used an ELISA design [@doi:10.3389/fmicb.2020.618097].
+Therefore, while analysis with ELISA requires laboratory, these results do suggest that ELISA achieves relatively high sensitivity, especially in the weeks following infection.
```
Comment (RLordan) [inline content/10.diagnostics.md:342]:
```suggestion
Therefore, while analysis with ELISA requires laboratory support and equipment, these results do suggest that ELISA achieves relatively high sensitivity, especially in the weeks following infection.
```

File: content/10.diagnostics.md [hunk 2]
```diff
@@ -331,18 +331,56 @@ Further research is required to determine the full extent to which sustained imm
...
-The authors are now working to get the assay into clinical use [@url:https://www.livescience.com/coronavirus-tests-available.html].
+LFIs are often available at point of care but can have very low sensitivity [@doi:10.1128/JCM.01361-20].
+
+#### Neutralizing Antibody Assays
+
+Neutralizing antibody assays play a functional role in understanding immunity that distinguishes them from other serological tests.
+The tests described above are all binding antibody tests.
+On the other hand, rather than simply binding an antibody to facilitate detection, neutralizing antibody determines whether an antibody response is present that would prevent infection [@doi:10.1128/Spectrum.01202-21; @doi:10.1038/s41467-020-17892-0].
+Therefore, these tests serve the purpose of evaluating the extent to which a sample donor has acquired immunity that will reduce susceptibility to SARS-CoV-2.
+As a result, neutralizing antibody assays have been used widely to characterize the duration of immunity following infection and to assess vaccine candidates.
```
Comment (RLordan) [inline content/10.diagnostics.md:375]:
```suggestion
As a result, neutralizing antibody assays have been used widely to characterize the duration of immunity following infection, to assess vaccine candidates, and to establish correlates of protection against infection and disease [@doi:10.1038/s41591-021-01377-8; @doi:10.1128/JCM.02107-20; @doi:gnx76z]
```

Comment (RLordan) [reply content/10.diagnostics.md:375]: In case I did the last one wrong: https://doi.org/10.1016/S2666-5247(21)00267-6

**Review by RLordan — APPROVED** [review]

File: content/10.diagnostics.md (hunk 2 above)
Comment (rando2) [reply content/10.diagnostics.md:375]: @RLordan Close enough that I was able to figure it out! It's the string of letters/numbers with a 10/ in front!

**Pull request merged**